frontend/util/ImageWithFallback.js

```diff
@@ -17,7 +17,9 @@ const ImageWithFallback = ({ image, imageStyle }) => {
     require('../assets/influencer10.jpg'),
   ];
   useEffect(()=>{
-    if(!image){
+    if(!image || image == null || image == undefined){
```

Comment:
why if-else if we are calling the same `handleError`

Reply:
Why because it has different types are coming from db. and it rising to app crash in mobile. Above if is for image doesn't exist ot equal to null or undefined. if else is for if it retuns number of some other. form for string which image is not displayed.